src/components/ChatGhostUI/ContentLoadingWrapper/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ContentLoader from 'react-content-loader';
+
+const propTypes = {
+    /* Children to wrap within the Content Loader */
+    children: PropTypes.node.isRequired,
+
+    /* Height of the content-loader */
+    height: PropTypes.number.isRequired,
+};
+
+const ContentLoadingWrapper = (props) => {
```

Comment:
**suggestion:** Get rid of this component entirely and merge it into the "one component" that will render various counts of lines.